tests/unit/conftest.py

```diff
@@ -0,0 +1,248 @@
+        "2021-09-27",
+        "2021-09-28",
+        "2021-09-29",
+        "2021-09-30",
+        "2021-10-01",
+        "2021-10-02",
+        "2021-10-03",
+        "2021-10-04",
+        "2021-10-05",
+        "2021-10-06",
+        "2021-10-07",
+        "2021-10-08",
+        "2021-10-09",
+        "2021-10-10",
+        "2021-10-11",
+        "2021-10-12",
+        "2021-10-13",
+        "2021-10-14",
+        "2021-10-15",
+        "2021-10-16",
+        "2021-10-17",
+        "2021-10-18",
+        "2021-10-19",
+        "2021-10-20",
+        "2021-10-21",
+        "2021-10-22",
+        "2021-10-23",
+        "2021-10-24",
+        "2021-10-25",
+        "2021-10-26",
+        "2021-10-27",
+        "2021-10-28",
+        "2021-10-29",
+        "2021-10-30",
+        "2021-10-31",
+        "2021-11-01",
+        "2021-11-02",
+        "2021-11-03",
+        "2021-11-04",
+        "2021-11-05",
+        "2021-11-06",
+        "2021-11-07",
+        "2021-11-08",
+        "2021-11-09",
+        "2021-11-10",
+        "2021-11-11",
+        "2021-11-12",
+        "2021-11-13",
+        "2021-11-14",
+        "2021-11-15",
+        "2021-11-16",
+        "2021-11-17",
+        "2021-11-18",
+        "2021-11-19",
+        "2021-11-20",
+        "2021-11-21",
+        "2021-11-22",
+        "2021-11-23",
+        "2021-11-24",
+        "2021-11-25",
+        "2021-11-26",
+        "2021-11-27",
+        "2021-11-28",
+        "2021-11-29",
+        "2021-11-30",
+        "2021-12-01",
+        "2021-12-02",
+        "2021-12-03",
+        "2021-12-04",
+        "2021-12-05",
+        "2021-12-06",
+        "2021-12-07",
+        "2021-12-08",
+        "2021-12-09",
+        "2021-12-10",
+        "2021-12-11",
+        "2021-12-12",
+        "2021-12-13",
+        "2021-12-14",
+        "2021-12-15",
+        "2021-12-16",
+        "2021-12-17",
+        "2021-12-18",
+        "2021-12-19",
+        "2021-12-20",
+        "2021-12-21",
+        "2021-12-22",
+        "2021-12-23",
+        "2021-12-24",
+        "2021-12-25",
+        "2021-12-26",
+        "2021-12-27",
+        "2021-12-28",
+        "2021-12-29",
+        "2021-12-30",
+        "2021-12-31",
+        "2022-01-01",
+        "2022-01-02",
+        "2022-01-03",
+        "2022-01-04",
+        "2022-01-05",
+        "2022-01-06",
+        "2022-01-07",
+        "2022-01-08",
+        "2022-01-09",
+        "2022-01-10",
+        "2022-01-11",
+        "2022-01-12",
+        "2022-01-13",
+        "2022-01-14",
+        "2022-01-15",
+        "2022-01-16",
+        "2022-01-17",
+        "2022-01-18",
+        "2022-01-19",
+        "2022-01-20",
+        "2022-01-21",
+        "2022-01-22",
+        "2022-01-23",
+        "2022-01-24",
+        "2022-01-25",
+        "2022-01-26",
+        "2022-01-27",
+        "2022-01-28",
+        "2022-01-29",
+        "2022-01-30",
+        "2022-01-31",
+    ],
+    dtype="datetime64[ns]",
+)
+
+
+complex_update_times = np.array(
```

Comment:
This could be written as a mix of ranges and individual dates to make it clearer where the breaks and irregularity lie. But I’m not fussed if we want to do that another time.